### DISCUSSION TITLE
全球已經把 AI 安全發展當成軍備競賽，台灣跟上了嗎？

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#504] [閒聊八卦]
回顧 2021 年，除了疫情、晶片和供應鏈，資訊安全也是產業界關注的重點。一方面因為多家知名企業網路系統被駭客入侵或綁架，造成影響範圍不同的傷害；另一方面，政府也開始積極進行規範，如金管會宣布 2022 年第一季前，符合一定條件的證券商、期貨、投信投顧與銀行業，須設立副總級以上資安長。

資訊安全（Cyber Security）的重要性已是共識，那麼逐漸邁向落地應用的 AI 呢？如何確保使用 AI 時的資訊安全，以避免造成對生命、財產或是整個社會的傷害？

看看這篇文章，歡迎分享對未來資安發展的看法吧

https://buzzorange.com/techorange/2022/03/02/ai-security-foxconn-next-forum/

**Comment by sean1325418** [comment]
AI技術這麼氾濫確實需要有一些管制，不然有心人可以輕易收集一堆資料。

**Comment by ThomasMeng0515** [comment]
沒辦法，台灣資工領域不太注重資安的領域，公司也不會花大錢請資安專家，所以不用期望台灣在這方面會有所進步。

**Comment by ghost** [comment]
說真的資安的建立往往比不上破解的速度，只能說能強則強

**Comment by hapiwangy** [comment]
資訊安全對於科技的發展的確是很重要的一環，攻擊和被攻擊方兩者都在不斷的進步。期望能夠有更多的人才投入到這方面的領域之中。

**Comment by HuanYuuuuuu** [comment]
台灣的AI技術還遠遠不及其他國家 更何況是資安的部分

**Comment by xkllkx** [comment]
台灣畢竟沒有很重視資訊業，不然台灣蠻適合發展軟實力的，不受到原料、土地等限制。

**Comment by Yuhsin000** [comment]
資安教育的方面好像也還沒跟上

**Comment by PoooYen** [comment]
其實台灣目前的教育資源跟社會資源還是在硬體方面，但是未來視軟體的世界，我們應該要好好重視...

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望我們能盡快找到之間的平衡。

**Comment by Sazmafuji** [comment]
台灣能夠在軟體方面強，萬能
